### ISSUE TITLE
Add to CAT list

### ISSUE THREAD

**Issue by lizliz** [TeaspoonTDA/teaspoon#114]
Teaspoon should be on this list, I think we just have to submit a pull request to them or something? https://cat-list.github.io/

I looked at it a while ago but forgot if/where I stopped

**Comment by stanweer1** [comment]
Yeah, have to submit a pull request / issue for adding a teaspoon.md file in their softwares folder. I made one (attached) but can't find many tags in their [already-defined tags list](https://github.com/cat-list/cat-list.github.io/blob/main/metadata/all-tags.txt) which are appropriate for tsp--we can make new ones though. 

[teaspoon.md](https://github.com/user-attachments/files/16196391/teaspoon.md)

**Comment by lizliz** [comment]
I just made a [pull request](https://github.com/cat-list/cat-list.github.io/pull/6), we can always update later. Looking at other repos they don't tend to have a paper listed, so as long as we have that listed prominently on our website we should be good.
